themes/openSUSE/src/submenu.inc

```diff
@@ -0,0 +1,122 @@
+% - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+%
+% Submenu handling.
+%
+% - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+% submenu entries are entries that have a label starting with '->FOO ' (note
+% the space) where FOO is an arbitrary string identifying the submenu.
+%
+% Menu entries that have an 'append' entry '->BAR' link to submenu BAR.
+%
+% As there are spaces in labels required to make this work, use the 'menu
+% label' statement in syslinux.
+
+% submenu entries must start with this string, followed by an arbitrary identifier
+/submenu.tag "->" def
+
+% submenues are stacked here
+/submenu.list 8 array def
```

Comment:
I am also fine with such limit, I just think we should document it somewhere else, not just burden it inside code.